test/unit/lib/censorPartnership.js

```diff
@@ -0,0 +1,304 @@
+'use strict';
+const { censorPartnership } = require('../../../src/lib/censorRecords');
+const roleWithFullAccess = ['full-details'];
+const roleWithoutFullAccess = ['births'];
+
+const partnershipData = {
+  id: 3000,
+    dateOfPartnership: "2020-09-23",
+  placeOfPartnership: {
+  address: "Address of Place of Partnership",
+    short: false
```

Comment:
Funny indentation.

`short` should be a string. (Can be just "Place of Partnsership".)

Reply:
fixed